common/src/sql/dbinit.sql

```diff
@@ -370,6 +370,26 @@ CREATE INDEX ON omicron.public.disk (
 ) WHERE
     time_deleted IS NULL AND attach_instance_id IS NOT NULL;
 
+CREATE TABLE omicron.public.image (
```

Comment:
RFD 4 mentions the following fields which are not yet stored here (nor a part of the API):
- version
- digest
- status